converters/jsonldToManifest.js

```diff
@@ -8,7 +8,7 @@
  * http://polymer.github.io/PATENTS.txt
  */
 
-var supportedTypes = ["Text", "URL", "Number", "Boolean"];
+var supportedTypes = ["Text", "URL", "Number", "Boolean", "Object"];
```

Comment:
don't need this here as Object isn't a schema.org type.

Reply:
done